src/providers/test/json-rpc-provider/get-code.test.ts

```diff
@@ -46,14 +46,12 @@ const provider = new JsonRpcProvider(rpcUrl);
 
 async function testGetCode(input: InputType, mockResult: string) {
   const spy = jest.spyOn(unfetch, 'default');
-  spy.mockImplementationOnce(() =>
-    Promise.resolve({
-      text: () =>
-        Promise.resolve(
-          JSON.stringify({ jsonrpc: '2.0', id: 1, result: mockResult }),
-        ),
-    } as Response),
-  );
+  spy.mockResolvedValueOnce({
```

Comment:
Change requested by the linter
![Screen Shot 2023-05-24 at 6 35 56 PM](https://github.com/dawsbot/essential-eth/assets/106350168/a1f2a573-9399-49b5-9419-0f1c99e6c71f)